### PR TITLE
Update documentation of upgrade inhibition

### PR DESCRIPTION
**Pull request by vojtechsokol**

### CODE REVIEW

File: docs/howto-inhibit-upgrade.md [hunk 1]
```diff
@@ -1,10 +1,19 @@
 # Upgrade Inhibition
 ### Process Inhibition
-With latest changes on Leapp and with new actors added to the el7toel8 Leapp repository, any actor can inhibit the upgrade process by producing a specific message when a problem is found. The message model to use in this case is [Inhibitor](https://github.com/oamg/leapp-repository/blob/master/repos/system_upgrade/el7toel8/models/reports.py). If there is at least one Inhibitor message produced before the Report phase, the upgrade will be stopped in the Reports phase, in which the messages are being collected. It means that any Inhibitor message produced **after** the Report phase will have **no** inhibiting effect. The details mentioned in the Inhibitor messages will be part of the report available to the user to review.
+With latest changes on Leapp and with new actors added to the el7toel8 Leapp
+repository, any actor can inhibit the upgrade process by producing a specific
+message when a problem is found. The message model to use in this case is
+[Report](https://github.com/oamg/leapp/blob/master/leapp/reporting/__init__.py).
+If there is at least one Report message produced before the Report phase,
```
Comment (jmikovic) [inline docs/howto-inhibit-upgrade.md]: I think there has to be Report with 'inhibitor' flag for upgrade to be stopped

File: docs/howto-inhibit-upgrade.md [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
+the upgrade will be stopped in the Reports phase, in which the messages are
+being collected. It means that any Report message produced **after** the
+Report phase will have **no** inhibiting effect. The details mentioned in the
```
Comment (jmikovic) [inline docs/howto-inhibit-upgrade.md]:
```suggestion
Report phase will **not** have inhibiting effect. The details mentioned in the
```

File: docs/howto-inhibit-upgrade.md [hunk 3]
```diff
@@ -25,82 +40,127 @@ name = 'check_system_arch'
...
+```
 
-After all the system checks are executed by different actors, an existing actor named [VerifyCheckResults](https://github.com/oamg/leapp-repository/tree/master/repos/system_upgrade/el7toel8/actors/verifycheckresults) is scheduled to run in the Leapp upgrade workflow. If some [Inhibitor](https://github.com/oamg/leapp-repository/blob/master/repos/system_upgrade/el7toel8/models/reports.py) message was generated by some previous execution of another actor in any previous phase of the workflow, like the sample one we just wrote, the following output will be displayed to the user:
+This is all that an actor needs to do in order to verify if some condition is
+present on the system and inhibit the upgrade process based on that check.
+
+After all the system checks are executed by different actors, an existing actor
+named [VerifyCheckResults](https://github.com/oamg/leapp-repository/tree/master/repos/system_upgrade/el7toel8/actors/verifycheckresults)
+is scheduled to run in the Leapp upgrade workflow. If some [Report](https://github.com/oamg/leapp/blob/master/leapp/reporting/__init__.py)
+message was generated by some previous execution of another actor in any
```
Comment (jmikovic) [inline docs/howto-inhibit-upgrade.md]: again missing 'inhibitor' flag

File: docs/howto-inhibit-upgrade.md [hunk 4]
```diff
@@ -25,82 +41,127 @@ name = 'check_system_arch'
...
+}
+```
 
-After all the system checks are executed by different actors, an existing actor named [VerifyCheckResults](https://github.com/oamg/leapp-repository/tree/master/repos/system_upgrade/el7toel8/actors/verifycheckresults) is scheduled to run in the Leapp upgrade workflow. If some [Inhibitor](https://github.com/oamg/leapp-repository/blob/master/repos/system_upgrade/el7toel8/models/reports.py) message was generated by some previous execution of another actor in any previous phase of the workflow, like the sample one we just wrote, the following output will be displayed to the user:
+This is all that an actor needs to do in order to verify if some condition is
+present on the system and inhibit the upgrade process based on that check.
+
+After all the system checks are executed by different actors, an existing actor
+named [VerifyCheckResults](https://github.com/oamg/leapp-repository/tree/master/repos/system_upgrade/el7toel8/actors/verifycheckresults)
+is scheduled to run in the Leapp upgrade workflow. If some [Report](https://github.com/oamg/leapp/blob/master/leapp/reporting/__init__.py)
```
Comment (bocekm) [inline docs/howto-inhibit-upgrade.md]: Link to https://leapp.readthedocs.io/en/latest/pydoc/leapp.reporting.html#leapp.reporting.Report instead.

File: docs/howto-inhibit-upgrade.md [hunk 5]
```diff
@@ -25,82 +41,127 @@ name = 'check_system_arch'
...
+2019-04-16 15:36:54.970 INFO     PID: 7455 leapp.workflow: Workflow interrupted due to the FailPhase error policy
 
 ============================================================
                         ERRORS
 ============================================================
 
-2018-09-25 19:30:49.989122 [ERROR] Actor: check_system_arch Message: Error: Fail: Unsupported arch!
-2018-09-25 19:30:49.989271 [ERROR] Actor: verify_check_results Message: Ending process due to errors found during checks
+2019-04-16 15:36:54.871634 [ERROR] Actor: verify_check_results Message: Unsupported arch
+2019-04-16 15:36:54.888818 [ERROR] Actor: verify_check_results Message: Ending process due to errors found during checks, see /tmp/leapp-report.txt for detailed report.
```
Comment (bocekm) [inline docs/howto-inhibit-upgrade.md]: Update the path to `/var/log/leapp-report.txt`.

**Pull request merged**